docs/catalog/actions/customising.rst

```diff
@@ -0,0 +1,9 @@
+Customising
```

Comment:
I'm not sure this belongs in it's own category. I would rather put it in `Miscellaneous` for now.

If we are going to put it in a separate category, then `Customizing` is confusing IMO. Perhaps we can call it `Message Formatting` or something like that.

Reply:
I think it will disappear under Miscellaneous
I'll change to Message Formatting